assets/js/_main.js

```diff
@@ -69,4 +69,12 @@ $(document).ready(function(){
     midClick: true // allow opening popup on middle mouse click. Always set it to true if you don't provide alternative source.
   });
 
+  // Change links on Book-reports Index page from amazon link to post url

```

Comment:
Can we elaborate a bit on this comment? I don't understand what this means.

Reply:
Yeah, that wasn't very good/descriptive reading that back now... good catch.

Basically what this is doing is changing the links around the images of the books on the book report **index page** so that they aren't pointing to the external (currently both books have Amazon links...) link, but instead linking to the actual book report page.  On the book report page, the link will still point to the external URL (Amazon currently...).  So, this only changes the link for navigation purposes on the index page.  This is how the current book report index page works as well.  I know you aren't married to that necessarily, but it seemed to make sense to me so liked to keep that in there?  Can certainly take it out completely if you are okay with the links around those images linking to Amazon even on the book report index page...

Does that make sense?  Screen capture below to help illustrate as well the links the js is changing dynamically:
<img width="1349" alt="screen shot 2018-10-13 at 10 57 11 pm" src="https://user-images.githubusercontent.com/2396774/46912183-65efc580-cf3c-11e8-8102-1b86f8495816.png">

I updated the comment to say "Loop through the image links on the book-reports index page and change them from pointing to external link to pointing to the individual book report page associated with that image." in aacec1cfb4e333c43d9cfb15464fc84077068c10.

Is that better/more descriptive?  Any thoughts on how better to describe it?